src/routes/base.routes.js

```diff
@@ -89,20 +92,20 @@ export class BaseRoutes {
    * @return {object}
    */
   create() {
-    this._validateControllerHandler('create')
+    this[ Symbol.for( 'validateControllerHandler' ) ]( 'create' );
```

Comment:
As a best practice, so the derived classes do not "know" about it to override it by mistake, or even use introspection to see it since getOwnPropertyNames do not return Symbols. Maybe just `Symbol` is even better actually.